staging/src/k8s.io/client-go/applyconfigurations/certificates/v1/certificatesigningrequeststatus.go

```diff
@@ -44,12 +44,10 @@ func (b *CertificateSigningRequestStatusApplyConfiguration) WithConditions(value
 	return b
 }
 
-// WithCertificate adds the given value to the Certificate field in the declarative configuration
-// and returns the receiver, so that objects can be build by chaining "With" function invocations.
-// If called multiple times, values provided by each call will be appended to the Certificate field.
-func (b *CertificateSigningRequestStatusApplyConfiguration) WithCertificate(values ...byte) *CertificateSigningRequestStatusApplyConfiguration {
-	for i := range values {
-		b.Certificate = append(b.Certificate, values[i])
-	}
+// WithCertificate sets the Certificate field in the declarative configuration to the given value
+// and returns the receiver, so that objects can be built by chaining "With" function invocations.
+// If called multiple times, the Certificate field is set to the value of the last call.
+func (b *CertificateSigningRequestStatusApplyConfiguration) WithCertificate(value []byte) *CertificateSigningRequestStatusApplyConfiguration {
+	b.Certificate = value
```

Comment:
The old implementation copied the input. Since maps & slices are references I'm wondering if we shouldn't take the argument as we do for scalars

Reply:
> take the argument as we do for scalars

What do you mean by that? Can you show an example?
I agree that not copying isn't great.

Reply:
> What do you mean by that? Can you show an example?

I only meant that for scalars we do things such as `obj.Field = argument`. This doesn't work the same for references

Reply:
Changed the code to use `copy`.